### PR TITLE
[Validator] added alpha3 option to country constraint

### PR DESCRIPTION
**Pull request by maxperrimond**
Documentation part of https://github.com/symfony/symfony/pull/35116

### CODE REVIEW

**Review by mazux — APPROVED** [review]

File: reference/constraints/Country.rst [hunk 1]
```diff
@@ -78,6 +79,15 @@ Options
 
 .. include:: /reference/constraints/_groups-option.rst.inc
 
+alpha3
+~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+If this option is ``true``, then the constraint will valid the value as
```
Comment (noniagriconomie) [inline reference/constraints/Country.rst]: I would write in the reverse way.
Default alpha2 description
And if set to true, validate against alpha3

Wdyt?

Comment (maxperrimond) [reply reference/constraints/Country.rst]: I don't mind but I wanted to be consistent with current documentation about `alpha3` option like in country and language type in form component.

File: reference/constraints/Country.rst [hunk 2]
```diff
@@ -7,6 +7,7 @@ Validates that a value is a valid `ISO 3166-1 alpha-2`_ country code.
 Applies to  :ref:`property or method <validation-property-target>`
 Options     - `groups`_
             - `message`_
+            - `alpha3`_
```
Comment (HeahDude) [inline reference/constraints/Country.rst]: We should keep an alpha order here

**Comment by wouterj** [comment]
Thanks for also taking care of the docs, @maxperrimond! Congratz on getting this merged in Symfony 5.1.

Fyi, I've applied @HeahDude's comment and added a directive indicating this feature is new in d0e01b6.

**Pull request merged**